### PR TITLE
Add sword challenge feature + update unit tests

### PR DESCRIPTION
**Pull request by FloreGiraudier**

### CODE REVIEW

File: src/Score/computeRoundState_test/sword-challenge.js [hunk 1]
```diff
@@ -7,11 +7,11 @@ import {
   SYMBOL_COIN,
 } from "/src/symbols/symbol-types.js"
 import { CARD_SWORD_CHALLENGE } from "src/Cards/card-types.js"
-import { computeScore } from "../ScoreHelpers.js"
+import { computeRoundState } from "../ScoreHelpers.js"
 
 // 8 coin
 {
-  const actual = computeScore({
+  const actual = computeRoundState({
```
Comment (dmail) [inline src/Score/computeRoundState_test/sword-challenge.js]: we should add a test ensuring 3 skulls or more means we get a negative score and not 0.

Comment (FloreGiraudier) [reply src/Score/computeRoundState_test/sword-challenge.js]: Fixed in the last commit :+1:

**Comment by dmail** [comment]
Be careful I have rebased this branch on master.
